diagnostic.md

```diff
@@ -14,6 +15,7 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    ember g route boston
```

Comment:
We need to include campus/ before boston so:
`ember generate route campus/boston`